src/app/services/itinerary.service.ts

```diff
@@ -0,0 +1,22 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { Itinerary } from '../models/itinerary';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ItineraryService {
+
+  constructor(private httpClient: HttpClient) { }
+
+  hostURL = 'http://localhost:3000/';
+
+  public lista(): Observable<Itinerary[]> {
+    var url=this.hostURL + 'itineraries'
+    var req=this.httpClient.get<Itinerary[]>(url);
+    req.subscribe(res=>console.log(res))
```

Comment:
Codacy has a fix for the issue: [Missing space before =>.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7122565)
```suggestion
    req.subscribe(res =>console.log(res))
```